wp-content/themes/core/theme.json

```diff
@@ -95,10 +95,100 @@
         }
       ]
     },
+    "custom": {
+      "fontWeight": {
+        "regular": 400,
+        "bold": 700
+      }
+    },
     "layout": {
-      "contentSize": "80ch",
+      "contentSize": "680px",
       "wideSize": "1200px"
     },
+    "typography": {
+      "customFontSize": false,
+      "dropCap": false,
+      "fluid": false,
```

Comment:
WP's fluid typography is disabled because we want more control over the clamp values